assets/src/util/assignment.js

```diff
@@ -21,15 +21,11 @@ const setAssigmentGoalInputState = (key, assignments, inputFocus) => {
 }
 
 const setAssignmentGoalGrade = (key, assignments, goalGrade) => {
-  // Use decimal place of pointsPossible if it's a decimal; otherwise, round to nearest tenth
-  const placeToRoundTo = (String(assignments[key].pointsPossible).includes('.'))
-    ? getDecimalPlaceOfFloat(assignments[key].pointsPossible) : 1
-
   return [
     ...assignments.slice(0, key),
     {
       ...assignments[key],
-      goalGrade: goalGrade === '' ? '' : roundToXDecimals(Number(goalGrade), placeToRoundTo),
+      goalGrade: goalGrade === '' ? '' : roundToXDecimals(Number(goalGrade), 3),
```

Comment:
Just thinking/writing through this: if we only store goals to the the thousandth, but in some wacky case, the `pointsPossible` is ten thousandths, nothing would break because the `roundToXDecimals(some_num, 5)` call would still return a valid result. I think thousandths will be enough to store, since I've never seen that before (only hundredths, I think).

Reply:
Ya that's right. It wouldn't hurt to store a couple more decimals worth of precision - if this becomes an issue in the future we can revisit.